src/main/java/CLI/Main.java

```diff
@@ -375,4 +387,91 @@ private static void checkHistoryAddJobsAndDownload(Job job, boolean removeInputB
             downloader.run();
         }
     }
-}
+
+    public static boolean checkUpdate() throws URISyntaxException {
```

Comment:
@kmr-rohit Please don't throw Exceptions just by adding it to the method signature. It would be better to handle that exception using a try-catch statement. Please consider adding it.

Reply:
Considered try/catch block to handle Exceptions

Reply:
Okay :+1: